djangocms_versioning_filer/static/filer/js/addons/table-dropzone.js

```diff
@@ -0,0 +1,279 @@
+// #DROPZONE#
+// This script implements the dropzone settings
+'use strict';
+
+// as of Django 2.x we need to check where jQuery is
+var djQuery = window.$;
+
+if (django.jQuery) {
+    djQuery = django.jQuery;
+}
+
+// Custom dialog modal used for file validation
+MicroModal.init({
+  onShow: modal => console.info(`${modal.id} is shown`), // [1]
+  onClose: modal => console.info(`${modal.id} is hidden`), // [2]
+  openTrigger: 'data-custom-open', // [3]
```

Comment:
Not sure what the numbers are for, I guess their docs: 
```suggestion
  openTrigger: 'data-custom-open',
```